<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_de_fornecedores</class>
 <widget class="QDialog" name="cadastro_de_fornecedores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>751</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Dados Fornecedor</string>
    </attribute>
    <widget class="QPushButton" name="btn_salvar_forn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>420</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar Fornecedor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_novo_fornecedor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Novo Fornecedor</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>721</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Nome Fantasia</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_nome_fantasia"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>CNPJ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cnpj"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Razão Social</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_razao_social"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>CEP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cep">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Bairro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_bairro">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Endereço</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_endereco">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Número</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_numero">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Cidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cidade">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Contato do fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_nome">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_tel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Ramal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_ram">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_email">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Departamento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_cont_dep">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>btn_novo_fornecedor</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>btn_salvar_forn</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Histórico Pedidos</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Histórico Financeiro</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Consultar Fornecedores</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
